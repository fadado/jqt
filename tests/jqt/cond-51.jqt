{% .published//empty | $jqt %}
	<meta name='published' content='{{.published}}' />
{% end %}
{% .updated//empty %}
	<meta name='updated' content='{{.}}' />
{% end %}
