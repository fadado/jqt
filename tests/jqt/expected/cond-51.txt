<meta name='updated' content='2016-06-26' />

